flutter_app/lib/widgets/display_all_logs_widget.dart

```diff
@@ -0,0 +1,51 @@
+import 'package:flutter/material.dart';
+import 'package:imacs/modules/get_airside_logs.dart';
+import 'package:imacs/screens/log_displayer_screen.dart';
+
+class LogsList extends StatelessWidget {
+  const LogsList({
+    super.key,
+    required this.getAirsideLogs,
+  });
+
+  final GetAirsideLogs getAirsideLogs;
+
+  static Route _logDisplayerRoute(BuildContext context, Object? arguments) {
+    final String fileContent = arguments as String;
+    return MaterialPageRoute(
+      builder: (context) => LogDisplayerScreen(fileContext: fileContent),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SingleChildScrollView(
+      child: SizedBox(
+        width: 600,
+        child: AspectRatio(
+          aspectRatio: 4 / 3,
+          child: ListView.builder(
+            itemCount: getAirsideLogs.getFiles().length,
+            itemBuilder: (context, index) {
+              return Padding(
```

Comment:
Adding the `Padding` here will increase distance between tiles, if your intention was to make the tiles bigger, move the `Padding` to the widget in the tiles' title attr

Reply:
i just wanted to make the list of file names look clickable